source/docs/04-cmn.xml

```diff
@@ -574,6 +574,16 @@
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve" valid="true"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/cmn/cmn-sample-schulhoff.txt" parse="text"/></egXML>
                      </figure>
                   </p>
+                  <p>One more way of notating beams is to use the <att>beam</att> attribute found on <gi scheme="MEI">note</gi>. This attribute allows three values:</p>
+                  <list type="gloss">
+                     <label>i (initial)</label>
+                     <item>Marks the start of a beam</item>
+                     <label>m (medial)</label>
+                     <item>Marks a participant in a beam other than the first or last</item>
+                     <label>t (terminal)</label>
+                     <item>Marks the end of a beam</item>
+                  </list>
+                  <p>Unlike <gi scheme="MEI">beam</gi> and <gi scheme="MEI">beamSpan</gi>, <att>beam</att> is purely in the analytical domain, meaning it does not represent the visual rendering of a beam (indeed Verovio does not render it) but that the note in question is <em>under a beam</em>. It may be used in cases where analytical needs require a flatter structure, or when beams might not have well-defined starts or ends, or when it simplifies analysis to have beaming be a property of notes rather than a separate element.</p>
```

Comment:
I see no point in mentioning Verovio here.
```suggestion
                  <p>Unlike <gi scheme="MEI">beam</gi> and <gi scheme="MEI">beamSpan</gi>, <att>beam</att> is purely in the analytical domain, meaning it does not represent the visual rendering of a beam but that the note in question is <em>under a beam</em>. It may be used in cases where analytical needs require a flatter structure, or when beams might not have well-defined starts or ends, or when it simplifies analysis to have beaming be a property of notes rather than a separate element.</p>
```

Reply:
`@beam` is allowed on chord, rest, and space in addition to note.

Reply:
```<att>beam</att> is purely in the analytical domain, meaning it does not represent the visual rendering of a beam but that the note in question is <em>under a beam</em>```

I think I get what @kepper was trying to say here, but I'm not sure I completely agree with phrasing like "purely in the analytical domain" and "it doesn't represent the visual rendering." There is no reason why an `@beam` attribute can't represent a visual beam, and this seems to be an artificial restriction on its use. It's simply an alternative way of indicating a beam structure. MEI is full of these attribute / element alternates: `<tuplet>` or `@tuplet`; `<tie>` or `@tie`; `<slur>` or `@slur`; `@grace` or `<graceGrp>`. Even `@accid` and `<accid>`, `@artic` and `<artic>`, and `@syl` and `<syl>` are forms of this same pattern.

I'm also with @rettinghaus that there is no need to mention Verovio here.